pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -12,6 +12,10 @@ fetchSubmodules=
 builder=
 branchName=$NIX_PREFETCH_GIT_BRANCH_NAME
 
+# ENV params
+out=${out:-}
+http_proxy=${http_proxy:-}
```

Comment:
I usually prefer to write it as:

```
: ${out:=}
: ${http_proxy:=}
```

as this avoid repeating the variable name.